install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
i feel like this file is kinda unnecessary.. if a user can't type two commands then there's some other issues going on there :joy:

Reply:
meh, i like having the option to just run a script and walk away :stuck_out_tongue: 
ill move this file to the gist i guess

Reply:
well I agree it's convenient to have a script that runs this all for you, however, every npm package should be a case of just `npm i` and `npm start`. sure there are 2 folders you have to run this in but it's not exactly rocket science. Having an install script like this adds another file to maintain, which may get outdated and cause confusion.